examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>Sounds negative here. Maybe: "Each remote Job has an unique identifier, referred as the Job arn. You can always retrieve arns of your job from the Jobs tab of Amazon Braket console. It is also recommend to keep a copy of the Job arn to pull the result at a later time. "</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>